pkg/arvo/sys/zuse.hoon

```diff
@@ -2357,6 +2357,150 @@
   ++  blake
     ~%  %blake  ..part  ~
     |%
+    ++  blake3
+      =<
+        =<  hash  :: cuter API
+        =+  [cv=iv flags=0b0]
+        ^?  |%
+        ::
+        +$  output  ^output  :: re-export
+        ::
+        ++  keyed  |=(key=byts hash(cv dat.key, flags f-keyedhash))
+        ::
+        ++  hash
+          ~%  %hash  ..blake3  ~
```

Comment:
Is this ok? First I tried slapping `~/` on various combinations of cores, but I couldn't get it to match what vere expected. Must have mixed up the ordering or something, idk

Reply:
This one should be

```
~%  %hash  ..hash  ~
```